components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -38,6 +38,10 @@ class PlacesApi(path: String) : PlacesManager, AutoCloseable {
     private var handle: AtomicLong = AtomicLong(0)
     private var writeConn: PlacesWriterConnection
 
+    // for history metadata
+    private var writeConn2: PlacesConnection
```

Comment:
These two properties are used to reference connections created by `PlacesApi::new_connection` rather than instances of the kt `PlacesReaderConnection` or `PlacesWriterConnection` classes.